src/components/PopoverWidget.tsx

```diff
@@ -0,0 +1,57 @@
+import { Popover, Transition } from '@headlessui/react';
+import { useState } from 'react';
+import { usePopper } from 'react-popper';
+
+interface PopoverProps {
+  children: React.ReactElement;
+  content: React.ReactElement;
+  isShowOnHover?: boolean;
+}
+
+const PopoverWidget = ({ children, content, isShowOnHover = false }: PopoverProps) => {
```

Comment:
I think we can just call this Popover, or even HoverCard. That is what Antd and Radix does. Which do you prefer?

https://www.radix-ui.com/docs/primitives/components/hover-card

https://ant.design/components/popover/

Reply:
Sure, we can name it just 'Popover'
Sounds simple and better.